src/Config.php

```diff
@@ -228,6 +218,22 @@ private static function getTerminusScript()
     }
 
     /**
+     * Ensures that directory paths work in any system
```

Comment:
I think this might blow up Cygwin, but I haven't used Cygwin in many years and don't quite remember.